_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-20 12:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+author: Kyle Begovich
+---
+
+Even though it's only June, planning for HackIllinois 2019 is already well under way, and our Summer Team would love to offer you an inside look at some of our motivations and guiding principles.
+
+# Something Old
+
+The blog has already explored the event last year in a few retrospective pieces, but here we will begin focusing on how it all applies to next year. We are currently in the process of engaging mentors from last year to get their take on their weekend on campus: what they liked and what they would like changed. These mentors are the heart of our hacker experience, tons of fun to host, and exceptionally talented contributors in their field. If you didn't get to engage with the mentors at the event, we hope to make this opportunity more central to the hacker experience this year.
+
+HackIllinois 2018 also saw many engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge players in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should take advantage of, and we'll continue to develop these aspects of the event for the years to come.
```

Comment:
Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience.

This sentence sounds/feels kind of weird to me, do you think you could reword it? Maybe "Hackers are able to learn a lot from sponsors, students, and mentors that they would not have otherwise had the opportunity to experience."